src/modules/auth/clients/clients.scss

```diff
@@ -47,6 +47,12 @@ $svg-arrow: var(--svg-arrow);
 	position: absolute;
 }
 
+.client_secret_btn {
```

Comment:
@Aringoaway Dont use `snake_case` for class names. Stick with the css class naming convention. Please, change it everywhere you used it.